src/_cffi_src/openssl/x509_vfy.py

```diff
@@ -15,17 +15,25 @@
  * Note that the result is an opaque type.
  */
 typedef STACK_OF(ASN1_OBJECT) Cryptography_STACK_OF_ASN1_OBJECT;
+typedef STACK_OF(X509_OBJECT) Cryptography_STACK_OF_X509_OBJECT;
 """
 
 TYPES = """
 static const long Cryptography_HAS_102_VERIFICATION_ERROR_CODES;
 static const long Cryptography_HAS_102_VERIFICATION_PARAMS;
 static const long Cryptography_HAS_X509_V_FLAG_TRUSTED_FIRST;
 static const long Cryptography_HAS_X509_V_FLAG_PARTIAL_CHAIN;
+static const long Cryptography_X509_LU_X509;
+static const long Cryptography_X509_LU_CLR;
 
 typedef ... Cryptography_STACK_OF_ASN1_OBJECT;
+typedef ... Cryptography_STACK_OF_X509_OBJECT;
 
-typedef ... X509_STORE;
+typedef ... X509_OBJECT;
+typedef struct {
+    Cryptography_STACK_OF_X509_OBJECT * objs;
+    ...;
+} X509_STORE;
```

Comment:
This is opaque in 1.1.0. To get this data there is a function called `X509_STORE_get0_objects`, but we'll need to write a version that works with < 1.1.0 for our own use. Same style as we did in: https://github.com/pyca/cryptography/blob/master/src/_cffi_src/openssl/rsa.py#L90-L115